src/main/java/com/incentives/piggyback/partner/dto/PartnerOrderEntity.java

```diff
@@ -26,4 +27,7 @@
 	private Date createdDate;
 	private Date lastModifiedDate;
 	private Integer isActive;
+	private List<String> interestCategories;
```

Comment:
This field should not be required. You can read the orderCategory field which has the category of the order. Users who have interest in this orderCategory should receive offers.